src/components/EventSection.tsx

```diff
@@ -0,0 +1,22 @@
+import { getUpcomingEvents } from "@/utils/getUpcomingEvents";
+import { EventsList } from "./EventsList";
+
+interface EventSectionProps {
+  city: string;
+}
+
+export async function EventSection({ city }: EventSectionProps) {
+  const events = await getUpcomingEvents();
+  const cityEvents = events?.filter(event => event.city === city);
```

Comment:
Is there a possibility there will be a case mismatch with `city`?

Reply:
Let's pray there won't be 😆